phpunit/class-wp-rest-block-types-controller-test.php

```diff
@@ -114,6 +114,26 @@ public function test_get_item() {
 		$this->check_block_type_object( $block_type, $response->get_data(), $response->get_links() );
 	}
 
+	/**
+	 *
+	 */
+	public function test_get_item_with_styles() {
```

Comment:
It would be nice either to update this test to include styles in the block definition or create another test that ensures that styles are properly merged.

Reply:
Fixed